src/statistic/update.py

```diff
@@ -356,6 +356,12 @@ def _get_objects_and_count_of_occurrence_firmware_and_file_db(self, object_path,
         combined_result = sum_up_lists(result_firmwares, result_files)
         return combined_result
 
+    def _get_objects_and_count_of_occurrence_firmware_and_file_db_of_ips(self, object_path, unwind=False, match=None):
```

Comment:
duplicate code. You can add an optional additional parameter 'sum_up_function' to the function above, to avoid code redundancy